src/makeInfinityAdder.js

```diff
@@ -3,8 +3,23 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(x) {
```

Comment:
The parameter `x` is declared but never used in the function. If it's not needed, consider removing it to avoid confusion.